br/pkg/lightning/metric/metric.go

```diff
@@ -24,10 +24,18 @@ import (
 const (
 	// states used for the TableCounter labels
 	TableStatePending   = "pending"
-	TableStateWritten   = "written"
 	TableStateImported  = "imported"
 	TableStateCompleted = "completed"
 
+	BytesStateTotalRestore   = "total_restore" // total source data bytes needs to restore
+	BytesStateRestored       = "restored"      // source data bytes restored during restore engine
+	BytesStateRestoreWritten = "written"       // bytes written during restore engine
+	BytesStateImported       = "imported"      // bytes imported during import engine
```

Comment:
What is the difference between `TableStateXXX` and `ByteStateXXX`, why not just reusing `TableState`?

Reply:
`TableState` means the state of a table, but `BytesState` look a little strange to me.

Reply:
different counter uses  different state. previously BytesCounter uses TableState, that's wield, so i normalized to BytesStateXXX